Makefile

```diff
@@ -802,17 +827,24 @@ app: appclean version data/osx/AppIcon.icns $(APPTARGET)
 	cp -R data/motd $(APPDATADIR)
 	cp -R data/credits $(APPDATADIR)
 	cp -R data/title $(APPDATADIR)
-	# bundle libc++ to fix bad buggy version on osx 10.7
-	LIBCPP=$$(otool -L $(APPTARGET) | grep libc++ | sed -n 's/\(.*\.dylib\).*/\1/p') && cp $$LIBCPP $(APPRESOURCESDIR)/ && cp $$(otool -L $$LIBCPP | grep libc++abi | sed -n 's/\(.*\.dylib\).*/\1/p') $(APPRESOURCESDIR)/
```

Comment:
I don't have a strong opinion about supporting 10.7 or not. The current player survey suggest we have one player on 10.6 and everyone else is on 10.9+ . If the effort for adding support for 10.7 is not too large, it might be worth it. If it's like 2 hours or more of work, I would just forget about it.